src/main/java/codesquad/springcafe/controller/ArticleController.java

```diff
@@ -0,0 +1,48 @@
+package codesquad.springcafe.controller;
+
+import codesquad.springcafe.model.Article;
+import codesquad.springcafe.db.ArticleDatabase;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicLong;
+
+@Controller
+public class ArticleController {
+    private final ArticleDatabase articleDatabase;
+    private final AtomicLong sequence = new AtomicLong();
+
+    @Autowired
+    public ArticleController(ArticleDatabase articleDatabase){
+        this.articleDatabase = articleDatabase;
+    }
+
+    @PostMapping("/articles")
+    public String createPost(
+            @RequestParam String title,
+            @RequestParam String content
+    ) {
+        long seq = sequence.incrementAndGet();
+        Article article = new Article(seq, title, content);
```

Comment:
```suggestion
        long seq = sequence.incrementAndGet();
        Article article = new Article( title, content);
```
시퀀스를 콘트롤러보다는 article이 static 멤버로 가지고 있는게 좋지 않을까요?